iam/api-client/src/main/java/iam/snippets/SetServiceAccountPolicy.java

```diff
@@ -21,49 +21,50 @@
 import com.google.iam.v1.Policy;
 import com.google.iam.v1.SetIamPolicyRequest;
 import com.google.protobuf.FieldMask;
-
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.List;
 
 public class SetServiceAccountPolicy {
-    public static void main(String[] args) throws IOException {
-        // TODO(developer): Replace the variables before running the sample.
-        // TODO: Replace with your project ID.
-        String projectId = "your-project-id";
-        // TODO: Replace with your service account name.
-        String serviceAccount = "your-service-account";
-        // TODO: Replace with your policy, GetPolicy.getPolicy(projectId, serviceAccount).
-        Policy policy = Policy.newBuilder().build();
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace the variables before running the sample.
+    // TODO: Replace with your project ID.
+    String projectId = "your-project-id";
+    // TODO: Replace with your service account name.
+    String serviceAccount = "your-service-account";
+    // TODO: Replace with your policy, GetPolicy.getPolicy(projectId, serviceAccount).
+    Policy policy = Policy.newBuilder().build();
 
-        setServiceAccountPolicy(policy, projectId, serviceAccount);
-    }
+    setServiceAccountPolicy(policy, projectId, serviceAccount);
+  }
+
+  // Sets a service account's policy.
+  public static Policy setServiceAccountPolicy(Policy policy, String projectId,
+                                               String serviceAccount) throws IOException {
+
+    //Construct the service account email.
+    //You can modify the ".iam.gserviceaccount.com" to match the service account name in which
+    //you want to delete the key.
+    //See, https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=en#deleting
```

Comment:
```suggestion
    // Construct the service account email.
    // You can modify the ".iam.gserviceaccount.com" to match the service account name in which
    // you want to delete the key.
    // See, https://cloud.google.com/iam/docs/creating-managing-service-account-keys?hl=en#deleting
```

Make this change wherever applicable in all the snippets.